.github/workflows/update.yml

```diff
@@ -0,0 +1,26 @@
+name: Update Nixpkgs pin
+on:
+  workflow_dispatch: # ALlows triggering manually
+  schedule:
+    - cron: '47 14 * * *' # runs every day at 14:47 UTC (chosen somewhat randomly)
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: cachix/install-nix-action@v26
+
+      - name: update
+        run: |
+          nix-build -A pkgs.npins
+          result/bin/npins update nixpkgs
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v6
```

Comment:
I would have thought there was some other PR-opening action, but this one by some guy is the most used. Odd.

Reply:
I guess it's fairly standard to have third-party actions

Reply:
I expected it to have some sort of ... github imprimatur.